src/assets/data/site-summary.json

```diff
@@ -1 +1 @@
-{"usa":{"lastDate":1642982400000,"totalCases":71697688,"totalDeaths":867868,"twoWeekPercentChangeInCases":-0.050349592555248825,"twoWeekPercentChangeInDeaths":0.3912423454408187,"totalVaccinationsInitiated":250964433,"totalPopulation":331486822}}
+{"usa":{"lastDate":1642982400000,"totalCases":71697688,"totalDeaths":867868,"twoWeekPercentChangeInCases":-0.050349592555248825,"twoWeekPercentChangeInDeaths":0.3912423454408187,"twoWeekPercentChangeInHospitalizations":0.1313993298482107,"totalVaccinationsInitiated":250964433,"totalPopulation":331486822}}
```

Comment:
This file is only included for testing purposes. This will be removed once this PR is approved.